cookiecutter.json

```diff
@@ -1,6 +1,7 @@
 {
     "project_name": "My Project",
     "repo_name": "{{ cookiecutter.project_name|lower|replace(' ', '-') }}",
+    "rootdir_name": "{{ cookiecutter.project_name|lower|replace(' ', '-') }}-project",
```

Comment:
Wouldn't `"{{ cookiecutter.repo_name }}-project"` be easier to read and understand?